test/unit/auth/credential.spec.ts

```diff
@@ -440,6 +441,49 @@ describe('Credential', () => {
       expect((getApplicationDefault())).to.be.an.instanceof(RefreshTokenCredential);
     });
 
+    it('should return a RefreshTokenCredential with FIREBASE_EMULATOR_CREDENTIALS', () => {
+      const firebaseEmulatorCred = {
+        client_id: "fakeclientid",
+        client_secret: "fakeclientsecret",
+        refresh_token: "fakerefreshtoken",
+        type: "authorized_user",
+      };
+      delete process.env.GOOGLE_APPLICATION_CREDENTIALS;
+      process.env.FIREBASE_EMULATOR_CREDENTIALS = JSON.stringify(firebaseEmulatorCred);
+
+      const defaultCred = getApplicationDefault();
+      expect(defaultCred).to.be.an.instanceof(RefreshTokenCredential);
+      expect((defaultCred as RefreshTokenCredential).refreshToken.refreshToken).to.eql(firebaseEmulatorCred.refresh_token);
```

Comment:
```
expect(defaultCred).to.have.property('refreshToken').that.includes(firebaseEmulatorCred);
```

Then you can don't have to make `refreshToken` public either.

Reply:
Oh thanks!